assets/css/custom-props/common.css

```diff
@@ -62,3 +78,27 @@
     --content-gutter: 20px;
   }
 }
+
+.transition-all {
+  transition: var(--transition-all);
+}
+
+.transition-colors {
+  transition: var(--transition-colors);
+}
+
+.transition-opacity {
+  transition: var(--transition-opacity);
+}
+
+.transition-shadow {
+  transition: var(--transition-shadow);
+}
+
+.transition-transform {
+  transition: var(--transition-transform);
+}
+
+.transition-none {
+  transition-property: none;
+}
```

Comment:
Do we need the classes? We don't use them anywhere for now, right? So we can skip them and add them later if necessary. :)